runtime/local/Cargo.toml

```diff
@@ -57,6 +57,8 @@ frame-system-rpc-runtime-api = { git = "https://github.com/paritytech/substrate"
 pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27", default-features = false }
 
 # Astar pallets
+chain-extension-trait = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.27", default-features = false }
```

Comment:
Is this still needed now that composite chain-extensions are supported?

Reply:
I think this is still needed in order to have an implementation in an external crate.
This Trait helps to define our chain-extension impls